src/app/demo-app.component.ts

```diff
@@ -14,19 +14,19 @@ import { FormsModule } from '@angular/forms';
 })
 
 export class DemoAppComponent {
-	r = 120;
-	g = 120;
-	b = 120;
-	w = 175;
-	h = 175;
-	p = 0;
-	m = 0;
-	o = 1.0;
-
-	autoheight = true;
-	autowidth = false;
+	r = signal(120);
```

Comment:
This is what I mean is a breaking change. The user no longer is able to use plain variables and must convert them to signals for them to update.

Reply:
@czeckd we can still use plain variable, it is just for out convenient we converted to signal, but signal inputs still accepts plain values instead of signal, as I mentioned previously **Angular signal inputs accepts same values as decorator inputs**.

Reply:
@sheikalthaf — I have tried to use the library revision with the prior version of the demo code and it just hangs somewhere in zone.js. 

I may be missing something obvious here, but I've not gotten your changes in the update signals library to work with apps that currently used the library. I am not seeing what I am doing wrong. 

So, again my concern is that this pull request seems to be a breaking change for apps that already using it.

Reply:
The hang is with this line in the template:
```html
<svg-icon [src]="img[onImg]" [stretch]="stretch" [svgStyle]="getNgStyle()">
</svg-icon>
```
Specifically 
`[svgStyle]="getNgStyle()"`

Reply:
@czeckd yes, previously it is a method call, which will be called every time on change detection. Now with signal inputs need should avoid method calls. because method call with change the value on every change detection which will ended up in a infinite loop.

Note: method calls are bad practice in Angular we should avoid that

Reply:
@sheikalthaf  So, how do you write the `getNgStyle` method to work without converting the demo app to use signals? Since the demo app is using a 'bad practice' it is necessary to break the app, and likely other apps that are using the library, that are legacy and used to call the method on change detection.

Wrapping the `getNgStyle` method with `computed` gets rid of the infinite loop, but still the app doesn't work with decorator inputs and is thus broken with the library update. I'm trying to prevent that for users of the library, or at the very least, alert that the updated library will potential break code and explain how to fix the issues. 

I want to merge your code, but the problem is, from the library publisher perspective is, if a user updates the library with this change either they will get a hung or non-functional app unless they convert the app to the new method. This would be seen as a breaking change, which can be okay, but it needs to be clearly stated when the updated library is published and explained how to fix the user's code.

I've attached a decorator variable version of the demo app. Thank you.

[demo-app.component.zip](https://github.com/czeckd/angular-svg-icon/files/15504514/demo-app.component.zip)

Reply:
@czeckd this is not a problem created by us. Whichever library uses a signal inputs then method calls should be avoided. So i think we need to mention in the release note that, inputs are now signal inputs. We can provide some examples about the usage